test/algorithms/overlay/get_clusters.cpp

```diff
@@ -102,8 +102,8 @@ int test_main(int, char* [])
     test_get_clusters<ep>();
 
     // This constant relates to the threshold in get_clusters,
-    // which is now return T(1) (earlier it was 1000)
-    double const multiplier = 1.0 / 1000.0;
+    // which is now return T(100) (earlier it was 1000)
+    double const multiplier = 1.0 / 10.0;
```

Comment:
I don't understand. Maybe the comment needs to point to the source in get_clusters.hpp

Reply:
My question is: "should `multiplier` be a function of `T` e.g.  `multiplier = T / 1000.0;` instead of manually update the fraction every time you change the threshold?" Am I missing something in the context here?

Reply:
Changed these lines a bit, avoiding the multiplier and enhancing the comment
:heavy_check_mark:

Reply:
> My question is: "should `multiplier` be a function of `T` e.g. `multiplier = T / 1000.0;` instead of manually update the fraction every time you change the threshold?" Am I missing something in the context here?

Sorry, I noticed this answer only after my own reply.
But I changed it, without multiplier now. I hope the threshold won't change again. Is it fine like this?